js_modules/dagit/packages/core/src/runs/RunActionsMenu.tsx

```diff
@@ -132,7 +135,21 @@ export const RunActionsMenu: React.FC<{
               />
               <MenuItem
                 tagName="button"
-                text="View all tags"
+                text={
+                  <div style={{display: 'contents'}}>
+                    View all tags
+                    <Box
+                      flex={{
+                        justifyContent: 'center',
+                        alignItems: 'center',
+                        display: 'inline-flex',
+                      }}
+                      padding={{vertical: 4, horizontal: 8}}
+                    >
+                      <SlashShortcut>t</SlashShortcut>
+                    </Box>
+                  </div>
```

Comment:
Could this just be a `<Box flex={{direction: 'row', ...}}>` that contains the text and the `SlashShortcut`?

Reply:
This box is being used to `center` the T specifically.